conans/client/api/conan_api.py

```diff
@@ -23,10 +96,8 @@ def wrapper(api, *args, **kwargs):
         except EnvironmentError:
             old_curdir = None
         try:
-            # TODO: Removing ConanApp creation, should we make it different for 2.0?
-            # Also removing the logger call, maybe conan_command can handle it
-            cache = ClientCache(api.cache_folder, api.out)
-            with environment_append(cache.config.env_vars):
+            api.create_app()
```

Comment:
This was using a shortcut on purpose to avoid the creation of tons of things that won't be used.

Anyway, this is ok, I think we will eventually remove this.